C01/project-metadata/C01-ProjectId612-metadata.json

```diff
@@ -69,12 +69,44 @@
       }
     },
     "type": "Feature",
+    "place_name": "Taita–Taveta, Kenya",
```

Comment:
This is needed if we want to display a place name on "More Projects" cards and at the top of the project page.

<img width="1072" alt="Screen Shot 2022-04-15 at 4 00 01 PM" src="https://user-images.githubusercontent.com/9438198/163641742-1ca78a43-bc0b-4ea4-8bb4-cb575b1a50f3.png">